docs/dataflows.rst

```diff
@@ -0,0 +1,134 @@
+.. _data-flows:
+
+==========
+Data flows
+==========
+
+.. contents::
+   :local:
+
+
+Position data
+=============
+
+Position data is stored in the database in shard tables:
+
+* ``blue_shard_*``
+* ``cell_*``
+* ``wifi_shard_*``
```

Comment:
It may be useful to specify the range of those wildcards:

* blue_shard_*: hex 0 through f
* cell_*: gsm, lte, wcdma, area?
* wifi_shared_*: hex 0 through f

Reply:
Good idea!